spec/factories/user_emails.rb

```diff
@@ -7,9 +7,14 @@
   factory :user_email, class: User::Email.name do
     primary true
     email
+    confirmed_at { Time.zone.now }
 
     after(:build) do |user_email|
       user_email.user ||= build(:user, emails: [user_email], emails_count: 0)
     end
+
+    trait :unconfirmed do
```

Comment:
Yah, the user emails are always confirmed in all of our test cases. Actually the `:unconfirmed` trait is not very useful,  We probably won't use it in future. (Gem already tested this and the confirmation flow. )

How about just remove the `:unconfirmed` trait ?